src/TConsole.cpp

```diff
@@ -745,23 +745,9 @@ void TConsole::closeEvent( QCloseEvent *event )
 
         if( mpHost->mFORCE_SAVE_ON_EXIT )
         {
-            QString directory_xml = QDir::homePath()+"/.config/mudlet/profiles/"+profile_name+"/current";
-            QString filename_xml = directory_xml + "/"+QDateTime::currentDateTime().toString("dd-MM-yyyy#hh-mm-ss")+".xml";
-            QDir dir_xml;
-            if( ! dir_xml.exists( directory_xml ) )
-            {
-                dir_xml.mkpath( directory_xml );
-            }
-            QFile file_xml( filename_xml );
-            if( file_xml.open( QIODevice::WriteOnly ) )
-            {
-                mpHost->modulesToWrite.clear();
-                XMLexport writer( mpHost );
-                writer.exportHost( & file_xml );
-                file_xml.close();
-                mpHost->saveModules(0);
+            mpHost->modulesToWrite.clear();
+            mpHost->saveProfile();
```

Comment:
That's how it was before - I'm just replicating the behaviour consistently. If we'd like to change it, changing how the saving mechanism works is a fair, but a different topic to enabling the Lua API to save the profile, wouldn't it be?

I like to scope my PRs for one small, atomical purpose so they are easier to digest instead of doing 10 different but semi-related changes in one profile that take a week to discuss because we'll have 10 different conversations happening at once.